### DISCUSSION TITLE
High / Low tariff depending on location

### DISCUSSION THREAD

**Discussion by arjenfvellinga** [dsmrreader/dsmr-reader#1485] [Questions & answers / How-to's] [answered]
As I understand from https://www.enexis.nl/consument/aansluiting-en-meter/tarief/dubbeltarief the hours for the high and low tariff depend on where you are living. But I can't find a setting to set your location. So I don't know now if the number of low and high tariff is calculated correctly.

Is this a missing settings, or am I missing something?

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Als het goed is **regelt je slimme meter dit zelf**. Deze weet exact welk tarief er op het moment geldt en houdt daar (vier) losse meterstanden voor bij. 

Vandaag is dit bijvoorbeeld goed te zien, want er geldt in het weekend alleen laagtarief:

![Selection_001](https://user-images.githubusercontent.com/16581744/110244266-3226f780-7f5e-11eb-9d58-a1bbcb3f283b.png)

In je eigen telegram kun je het ook zien aan deze velden:

![Selection_002](https://user-images.githubusercontent.com/16581744/110244280-3e12b980-7f5e-11eb-8286-7f737f461bbf.png)

De bovenste vier zijn:
- Verbruikt tarief 1
- Teruggeleverd tarief 1
- Verbruikt tarief 2
- Teruggeleverd tarief 2

En de onderste:
- Huidig tarief (1)

In Nederland is tarief 1 laag en tarief 2 hoog.

**Comment by arjenfvellinga** [comment]
Ah duidelijk!

Dank voor je snelle antwoord!
